languages/rust/exercises/concept/tuples/.docs/instructions.md

```diff
@@ -0,0 +1,56 @@
+# Instructions
+
+In this exercise you'll explore tuples in Rust by building a kitchen calculator. Imagine you are following a sensible gingersnap cookie recipe but it uses American units for measuring ingredients. You are most familiar with metric units. Being an intrepid programmer, you want to automate that work by writing a kitchen calculator.
+
+Use the following chart in your program:
+
+| Unit to convert | volume | in millilitres (mL) |
+| --------------- | ------ | ------------------- |
+| mL              | 1      | 1                   |
+| US cup          | 1      | 240                 |
+| US fluid ounce  | 1      | 30                  |
+| US teaspoon     | 1      | 5                   |
+| US tablespoon   | 1      | 15                  |
+
+## 1. Extract the numeric component from a tuple describing volume
+
+- Given the following tuple `(Unit::USCup, 1)`, return the 2nd element.
```

Comment:
The instructions of a task should mention what method/function the student is to implement (see [this example](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/strings/.docs/instructions.md#1-get-message-from-a-log-line)). So maybe do something like this:

```suggestion
Implement the `get_volume` function that takes a unit/volume pair tuple as its parameter, and returns the volume part.
```

Or something like that.